apiserver/src/api.rs

```diff
@@ -149,6 +153,14 @@ pub async fn run_server<T: 'static + BottlerocketNodeClient>(
             futures::future::ready(())
         });
 
+    // Set up prometheus metrics
+    let exporter = opentelemetry_prometheus::exporter().init();
```

Comment:
I think we want an `Option<PrometheusExporter>` to be passed as an argument to this function, that way we can create the appropriate exporter in our telemetry initialization code in `main`.